index.html

```diff
@@ -8720,36 +8719,29 @@ <h3>Graph Containers</h3>
       </pre>
       <pre class="expanded"
            data-transform="updateExample"
-           data-result-for="Embedding JSON-LD in HTML with comments-original">
+           data-result-for="Embedding JSON-LD containing HTML in HTML-original">
       <!--
       [{
-        "@type": "http://schema.org/WebPageElement",
+        "@type": ["http://schema.org/WebPageElement"],
         "http://schema.org/name": [{"@value": "Encoding Issues"}],
         "http://schema.org/description": [
-          {"@value": "Issues list such as unescaped &lt;/script&gt; or --&gt;"}
+          {"@value": "Issues list such as unescaped </script> or -- >"}
```

Comment:
Similar to my other comment--would expansion of the raw JSON-LD result in this variation or would those retain the entities from the original?

Reply:
This "other comment" (since it's from a different repo's PR review) https://github.com/w3c/json-ld-api/commit/b969d200d5e399396d4089f1d41dccd0b3c43ab2#r31502674

Reply:
The original included JSON-LD in a script element with escaped entities. When extracted, and subsequently processed, the entities are resolved.

There's some trickery (`-- >` vs. `-->`) because of issues in the spec itself, as `--` can not appear in a comment, which it would otherwise. Code in both common.js and extract-examples explicitly manages this.